openapi/src/common/schemas.yaml

```diff
@@ -11,7 +11,7 @@ components:
     CloudPlatform:
       type: string
       description: Enum representing a cloud platform type.
-      enum: ['AZURE', 'GCP']
+      enum: ['GCP', 'AZURE']
```

Comment:
out of curiosity, why this swap?

Reply:
Every other place - GCP was listed before Azure. keeping it consistent.